File: Examples/DotNET/SDK/VisualBasic/Paragraph/GetFontOfRun.vb
Imports Aspose.Cloud

Namespace Paragraph
    Class GetFontOfRun
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"
            Dim subdirection As String = "Paragraph"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirection)

            ' Get Font Information of a Run from a Word Document
            Dim WordsFontResponse As WordsFontResponse = Common.WordsService.WordsParagraph.GetDocumentFontByParagraphAndRun(inputfile, 0, 0, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + WordsFontResponse.Font.Name + " font.")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Paragraph/GetFontOfRun.cs
using System;
using Aspose.Cloud;

namespace CSharp.Paragraph
{
    class GetFontOfRun
    {
        public static void Run()
        {
            string inputfile = "doc-sample.doc";
            string subdirection = "Paragraph";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirection);

            // Get Font Information of a Run from a Word Document
            WordsFontResponse WordsFontResponse = Common.WordsService.WordsParagraph.GetDocumentFontByParagraphAndRun(inputfile, 0, 0, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + WordsFontResponse.Font.Name + " font.");
        }
    }
}
